libs/shared/src/types/shared/index.ts

```diff
@@ -1 +1,5 @@
 export type CustomDataType = { [key: string]: string | string[] | boolean | number | undefined };
+
+export const NOVU_SUB_MASK = 'nvsk.';
+
+export type EncryptedSecret = `${typeof NOVU_SUB_MASK}${string}`;
```

Comment:
kudos to Richard for helping me to be more familiar with Template Literal Types.
This small type highlighted some potential issues we could have in compilation time while developing this feature, and will probably prevent future issues.